src/pages/PlanPage/subRoutes/CurrentOrgPlan/CurrentPlanCard/FreePlanCard/FreePlanCard.jsx

```diff
@@ -100,10 +112,21 @@ PlanUpgrade.propTypes = {
 }
 
 function FreePlanCard({ plan, scheduledPhase }) {
-  const { provider } = useParams()
-
+  const { provider, owner } = useParams()
+  const { codecovTrialMvp } = useFlags({
+    codecovTrialMvp: false,
+  })
   const { data: ownerData } = usePlanPageData()
+  const { data: planData } = usePlanData({
+    provider,
+    owner,
+    opts: { enabled: codecovTrialMvp },
+  })
+
   const uploadsNumber = ownerData?.numberOfUploads
+  const trialOngoing =
+    isTrialPlan(planData?.plan?.planName) &&
+    planData?.plan.trialStatus === TrialStatuses.ONGOING
```

Comment:
Could we add optional chaining here?

Reply:
Optional chaining isn't required here, because if `plan` exists then `trialStatus` will also exist as the value returned from the API is non-null